charts/sealed-secrets/crds/bitnami.com_sealedsecrets.yaml

```diff
@@ -89,14 +89,6 @@ spec:
                         additionalProperties:
                           type: string
                         type: object
-                      finalizers:
```

Comment:
This is 3rd party CRD that the operator relies on. We can just modify it because we can't control any side effects. Moreover these  CRD properties are perfectly fine.